maps/skins.html

```diff
@@ -4,14 +4,14 @@
 permalink: /maps-list/mods/
 ---
 <div class="maps-col-wrapper">
-  {%- for map in site.maps -%}
-  {%- if map.mapType == "MAP_SKIN" %}
+  {% for map in site.maps %}
+  {% if map.mapType == "MAP_SKIN" %}
     <div class="maps-col" data-map-name="{{ map.mapName | escape }}" data-map-name-slug="{{ map.slug }}">
       <div class="maps-col-content-wrapper">
         <h3 class="maps-col-item">{{ map.mapName | escape }}</h3>
         <p class="maps-col-item"><a class="maps-col-button button" href="{{ '/map/' | prepend: site.baseurl | append: map.slug }}/">View</a></p>
       </div>
     </div>
-  {%- endif -%}
-  {%- endfor %}
+  {% endif %}
+  {% endfor %}
```

Comment:
I believe you don't have the latest version of jekyll installed on your machine, this feature is available since 3.5 and saves a lot of whitespace